app/validators/qc_receptions_factory_validator.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# Failed validations return unprocessable_entity
+# These are being validated before any entity is created
+class QcReceptionsFactoryValidator < ActiveModel::Validator
```

Comment:
Just to double check. You have a validate_each method in this validator but there is an ActiveModel EachValidator?

Reply:
Refactored